packages/components/src/navigator/test/index.tsx

```diff
@@ -13,11 +13,13 @@ import { useState } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+import { Button } from '../../button';
```

Comment:
A more standard way of importing the `Button` in the package is

```suggestion
import Button from '../../button';
```